releasetasks/templates/firefox/enUS.yml.tmpl

```diff
@@ -157,12 +157,14 @@
                 {% if signing_class == "dep-signing" %}
                 MOZ_DISABLE_MAR_CERT_VERIFICATION: true
                 SIGNING_CERT: "dep"
-                {% else %}
+                {% elif signing_class == "nightly-signing" %}
+                SIGNING_CERT: "nightly"
+                {% elif signing_class == "release-signing" %}
                 SIGNING_CERT: "release"
                 {% endif %}
-                {% if branch == "mozilla-beta" %}
+                {% if accepted_mar_channel_id %}
```

Comment:
It's always defined, I've made the necessary changes in my WIP for tools under bug 1358611. But I guess you're right, I should add that additional check too, it doesn't hurt 👍